les/serverpool.go

```diff
@@ -96,6 +96,7 @@ var (
 	sfQueried          = serverPoolSetup.NewFlag("queried")
 	sfCanDial          = serverPoolSetup.NewFlag("canDial")
 	sfDialed           = serverPoolSetup.NewFlag("dialed")
```

Comment:
Now the `sfDialed` seems not a good name, let's rename is to `sfDialing`

Reply:
done